doc/error.md

```diff
@@ -1,20 +1,20 @@
 Error Handling
 ======================
 
-qc::conn_marshal
+[qc::error_handler](doc/procs/error_handler.md)
 --------------------------
 
-If ns_register_proc is used to register the [qc::conn_marshal](procs/conn_marshal.md) to deal with incoming requests, errors are by default passed to [qc::error_handler](procs/error_handler.md).
+Errors are classified by the errorCode returned and [qc::error_handler](doc/procs/error_handler.md) will return a suitable message and code to the client based upon the error code.
 
```

Comment:
Can you change errorCode -> error code.

I think we are trying to move away from using the global error variables errorMessage, errorCode & errorInfo. They are still supported for backwards compatibility though.